spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -628,6 +628,17 @@ function recvPacket(
     // do sequence check before any state changes
     if channel.order == ORDERED || channel.order == ORDERED_ALLOW_TIMEOUT {
         nextSequenceRecv = provableStore.get(nextSequenceRecvPath(packet.destPort, packet.destChannel))
+        if (packet.sequence < nextSequenceRecv) {
+          emitLogEntry("recvPacket", {
+            sequence: packet.sequence, 
+            timeoutHeight: packet.timeoutHeight, 
+            port: packet.destPort, 
+            channel: packet.destChannel,
+            timeoutTimestamp: packet.timeoutTimestamp, 
+            data: packet.data
```

Comment:
The fields emitted don't match the fields emitted by the implementation, i.e., https://github.com/cosmos/ibc-go/blob/c8e2ebcf2eaa6b487ca52499063c7c3db4253398/modules/core/04-channel/keeper/events.go#L157. There should be `data`, `timeoutHeight`, `timeoutTimestamp`, `sequence`, `srcPort`, `srcChannel`, `dstPort`, `dstChannel`, `ordering`, `connection`.